doc/source/release-notes/Percona-Server-5.7.10-2.rst

```diff
@@ -0,0 +1,78 @@
+
+ |TokuDB| status variables were missing from |Percona Server| :rn:`5.7.10-1`. Bug fixed :bug:`1527364` (:tokubug:`923`).
+
+ Attempting to rotate the audit log file would result in audit log file name :file:`foo.log.%u` (literally) instead of a numeric suffix. Bug fixed :bug:`1528603`.
+
+ Adding an index to an |InnoDB| temporary table while :variable:`expand_fast_index_creation` was enabled could lead to server assertion. Bug fixed :bug:`1529555`.
+
+ |TokuDB| would not be upgraded on *Debian*/*Ubuntu* distributions while performing an upgrade from |Percona Server| 5.6 to |Percona Server| 5.7 even if explicitly requested. Bug fixed :bug:`1533580`.
+
+ Server would assert when both |TokuDB| and |InnoDB| tables were used within one transaction on a replication slave which has binary log enabled and slave updates logging disabled. Bug fixed :bug:`1534249` (upstream bug :mysqlbug:`80053`).
+
+ `MeCab Full-Text Parser Plugin <https://dev.mysql.com/doc/refman/5.7/en/fulltext-search-mecab.html>`_ has not been included in the previous release. Bug fixed :bug:`1534617`.
+
+ Fixed server assertion caused by ``Performance Schema`` memory key mix-up in ``SET STATEMENT ... FOR ...`` statements. Bug fixed :bug:`1534874`.
+
+ Setting the :variable:`innodb_sched_priority_purge` (available only in debug builds) while purge threads were stopped would cause a server crash. Bug fixed :bug:`1368552`.
+
+ Enabling |TokuDB| with ``ps_tokudb_admin`` script inside the Docker container would cause an error due to insufficient privileges even when running as root. In order for this script to be used inside docker containers this error has been changed to a warning that a check is impossible. Bug fixed :bug:`1520890`. 
+
+ Write-heavy workload with a small buffer pool could lead to a deadlock when free buffers are exhausted. Bug fixed :bug:`1521905`.
+
+ |InnoDB| status will start printing negative values for spin rounds per wait, if the wait number, even though being accounted as a signed 64-bit integer, will not fit into a signed 32-bit integer. Bug fixed :bug:`1527160` (upstream :mysqlbug:`79703`).
+
+ |Percona Server| 5.7 couldn't be restarted after |TokuDB| has been installed with ``ps_tokudb_admin`` script. Bug fixed :bug:`1527535`.
+
+ Fixed memory leak when :variable:`utility_user` is enabled. Bug fixed :bug:`1530918`.
+
+ Page cleaner worker threads were not instrumented for ``Performance Schema``. Bug fixed :bug:`1532747` (upstream bug :mysqlbug:`79894`).
+
+ Busy server was preferring LRU flushing over flush list flushing too strongly which could lead to performance degradation. Bug fixed :bug:`1534114`.
+
+ :file:`libjemalloc.so.1` was missing from binary tarball. Bug fixed :bug:`1537129`.
+
+ When ``cmake/make/make_binary_distribution`` workflow was used to produce binary tarballs it would produce tarballs with ``mysql-...`` naming instead of ``percona-server-...``. Bug fixed :bug:`1540385`.
+
+ Bug fixed :tokubug:`917`.
+
+ Bug fixed :tokubug:`926`.
```

Comment:
5.7 changed the way a storage engine should report index statistics to the optimizer. This issues implements the new index statistic reporting within TokuDB. Following the InnoDB example, the default value for tokudb_cardinality_scale has changed from 50% to 100%. Implementing this addresses a server crash deep in the optimizer code that results from a combination of conditions.